model/clusters_mgmt/v1/cluster_operators_condition_type.model

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Overall state of a cluster operator
```

Comment:
Add a dot to end the sentence. It is good typing practice and in addition I plan to add a documentation generator that uses that dot to identify the first sentence in order to use it as the summary.